content/_index.md

```diff
@@ -3,50 +3,43 @@
 
 ### News
 
-- **Nordic-RSE conference 2026** will happen in **Tromsø, Norway on June 9-10 2026**. More info coming soon!
-- **Conference 2025**: Our second Nordic-RSE conference happened on May 20 + 21 2025 in Gothenburg, Sweden.
-  Blog posts coming soon! Thanks to everyone who participated!
+- One of our members [was at RSECon2025](/blog/rsecon-worldwide/) in UK and has brought back lots of motivation!
+- RSEs in Finlands capital region [met to celebrate International RSE Day](/blog/int-rse-day/) on October 9th!
+- The **Nordic-RSE conference 2026** will happen in Tromsø, Norway on June 9-10 2026. More info coming soon!
+- Return to the **Conference 2025**: Our [second Nordic-RSE conference](/blog/nrse-conference-report/) happened on May 20 + 21 2025 in Gothenburg, Sweden. Thanks to everyone who participated!
 
 
-### Who we are
+### What is Nordic-RSE
 
-A **Research Software Engineer (RSE)** is someone in between research
-and technology (software, computing, data).  Perhaps they work as a
-expert within a research group, or perhaps they work as a specialist
-in a service that supports other researchers.  Does this sound like
-you?  Read on...
+The **Nordic Research Software Engineers** association (in short, Nordic-RSE) 
+is a grass-root association that aims at providing a professional network to 
+**Research Software Engineers** (RSE) in the Nordics. While the term "RSE" is 
+not as of yet of common use here, Nordic-RSE is currently a community for 
+people who operate at the interface between research and digital technologies 
+(software, computing, data). We operate on an inclusivity base: if you think 
+that the above description is you, then you are welcome in our association!
 
-The **Nordic Research Software Engineers** association hopes to
-promote the careers of RSEs and the advancement of science by:
+As an association, Nordic-RSE works on promoting the careers of RSE by:
 
 * being a networking platform for specialized research supporters,
 * provide a community and professional development network for RSEs
   (**in particular supporting career development**),
 * organizing events to bring RSEs and related people together, and
-* support whose who want to bring RSE services and skills to their
+* support those who want to bring RSE services and skills to their
```

Comment:
Should be supporting (and providing two lines above) as a continuation of the "... by:" before the list.